### ISSUE TITLE
Default protocols and auto enrollment

### ISSUE THREAD

**Issue by michael-wynne-wsu** [MukurtuCMS/Mukurtu-CMS#266]
Do we want ANY kind of default protocol? Either within a community or within a site as a whole?

**Comment by michael-wynne-wsu** [comment]
@joelle-birano-wsu @christa-kavanagh-wsu With the addition of community>protocol enrolment workflows, do you think it's necessary to include auto-enrolment options?

**Comment by christa-kavanagh-wsu** [comment]
> [@joelle-birano-wsu](https://github.com/joelle-birano-wsu) [@christa-kavanagh-wsu](https://github.com/christa-kavanagh-wsu) With the addition of community>protocol enrolment workflows, do you think it's necessary to include auto-enrolment options?

I don't think it's necessary.

**Comment by michael-wynne-wsu** [comment]
I think there may be merit in exploring this post-release, so going to move it to that section.

**Comment by joelle-birano-wsu** [comment]
> [@joelle-birano-wsu](https://github.com/joelle-birano-wsu) [@christa-kavanagh-wsu](https://github.com/christa-kavanagh-wsu) With the addition of community>protocol enrolment workflows, do you think it's necessary to include auto-enrolment options?

As long as folks have to turn it on rather than have it on by default I think it can be useful. I think we can into that issue before where someone didn't know folks were being auto-added to a community or something?

**Comment by michael-wynne-wsu** [comment]
Yeah the v3 implementation was extremely flawed. I would start exploration from scratch, not try to replicate the v3 operation.

I also can't think of a way that it would work that wouldn't require several big warning messages, which makes me think it's best not to rush this for a release.
